default.nix

```
@@ -77,6 +41,27 @@ rec {
       in
       f "";
 
+    # Recursively evaluate attributes for an attribute set.
+    # Coupled with an evaluated nixos configuration, this presents an efficient
+    # way for checking module types.
+    evalAttrsRecursive =
```

The name is a bit misleading, because you only get the `true`s and a bit of structure, not the actual value. Maybe we want it to be more like

```suggestion
    foceEvalRecursive =
```